### ISSUE TITLE
Splitter: Accessibility updates

### ISSUE THREAD

**Issue by shawnharvell** [primefaces/primereact#6039]
### Describe the bug

There are a few small bugs in the Splitter component:

- role="separator" is on the wrong element, it should be on the gutter handle element where the other aria attributes are located (in particular, the aria-orientation attribute)
- the documentation indicates that the step property belongs to SplitterPanel, however when you look at the code it's clear that really it goes on Splitter itself, this is simply a bug in the typescript typing file that results in the documentation appearing incorrect
- if you use minSize on a SplitterPanel, when you try to use Home/End/Enter keys for maximize/minimize behavior, then the arrow keys stop working

I've actually coded fixes for this in my local cloned repo that seem to be working great BUT I am unable to push a branch to the repo to create a PR as GitHub says I do not have write access.  If I could be given that access I'd be glad to push and PR OR alternatively would be glad to somehow share the code changes I made with somone who can do it.

### PrimeReact version

10.5.1

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

**Comment by melloware** [comment]
you have to fork the repo.  You can't push directly to this repo you have to fork it, make changes in your fork and it will submit a PR from there.

See: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by shawnharvell** [comment]
thanks @melloware PR https://github.com/primefaces/primereact/pull/6040

**Comment by melloware** [comment]
Thanks it's great. I am going to make these same changes in PrimeFaces JSF.
